src/main/java/com/minenash/customhud/HudElements/supplier/StringSupplierElement.java

```diff
@@ -41,6 +41,7 @@ public class StringSupplierElement implements HudElement {
     public static final Supplier<String> TIME_AM_PM = () -> ComplexData.timeOfDay < 12000 ? "am" : "pm";
 
     public static final Supplier<String> FACING = () -> cameraEntity().getHorizontalFacing().getName();
+    public static final Supplier<String> FACING_SHORT = () -> FACING.get().substring(0, 1).toUpperCase();
```

Comment:
I don't personally like running a supplier in use of another one

Reply:
I'm on vacation right now, but will change it next week!